### PR TITLE
Adding shortcut paths and ngen tags to setup vsixes

### PR DESCRIPTION
**Pull request by jmarolf**
These tags were missing from the telemetry vsix and the setup.next vsix.

@brettfo @jasonmalinowski 
@dotnet/roslyn-infrastructure

### CODE REVIEW

**Comment by jmarolf** [comment]
test vsi please

File: src/VisualStudio/Telemetry/source.extension.vsixmanifest [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <Identity Id="28354cb8-c808-4138-bfce-33aa846bbd51" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
     <DisplayName>Roslyn Telemetry</DisplayName>
     <Description>Telemetry implementation for Roslyn atop Visual Studio.</Description>
+    <ShortcutPath>..\CommonExtensions\Microsoft\ManagedLanguages\VBCSharp\Telemetry</ShortcutPath>
```
Comment (jasonmalinowski) [inline src/VisualStudio/Telemetry/source.extension.vsixmanifest]: Why no ngen here?

**Comment by jasonmalinowski** [comment]
:+1: but one question about lack of ngen for one assembly.

File: src/VisualStudio/Telemetry/source.extension.vsixmanifest (hunk 1 above)
Comment (jmarolf) [inline src/VisualStudio/Telemetry/source.extension.vsixmanifest]: Do we currently ngen our telemetry?

**Pull request merged**